src/keymap.cpp

```diff
@@ -85,14 +85,28 @@ static const std::vector<OpDesc> opdescs = {
 		"next",
 		"J",
 		_("Go to next article"),
-		KM_FEEDLIST | KM_ARTICLELIST | KM_ARTICLE
+		KM_ARTICLELIST | KM_ARTICLE
+	},
+	{
+		OP_NEXT,
+		"next",
+		"J",
+		_("Go to next entry"),
+		KM_FEEDLIST | KM_DIALOGS | KM_DIRBROWSER | KM_FILEBROWSER | KM_FILTERSELECT | KM_TAGSELECT | KM_URLVIEW | KM_PODBOAT
```

Comment:
Same as docs: just unify all dialogs under a single entry with a doc saying "Go to next/previous entry".

Reply:
Done, thanks for the review!